crates/cxx-qt-build/Cargo.toml

```diff
@@ -24,6 +24,7 @@ codespan-reporting = "0.11"
 version_check = "0.9"
 serde = { version = "1.0", features = ["default", "derive"] }
 serde_json = "1.0"
+syn.workspace = true
```

Comment:
Aha i might not need that anymore now that i pass the include path as a PathBuf/String before i was building the extern "C++" block with parse_quote in cxx-qt-build and injecting it, so this can likely go...